wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -233,30 +244,36 @@ def _add_to_launch_queue(
         config: Optional[Dict[str, Any]] = None,
     ) -> "public.QueuedRun":
         """Add a launch job to the Launch RunQueue."""
-        run_id = run_id or generate_id()
-        # One of Job and URI is required
-        _job = self._kwargs.get("job", None)
-        _uri = self._kwargs.get("uri", None)
+        # One of Job and docker URI is required
+        _job = self._kwargs.get("job")
+        _uri = self._kwargs.get("launch", {}).get("image_uri")
```

Comment:
these keys should exist in the kwargs, but we could also backup to the sweep config here